geometry/densify-and-generalize/README.md

```diff
@@ -1,35 +1,35 @@
-# Densify and Generalize
+# Densify and generalize 
 
-Modify a polyline while preserving its general shape.
+A multipart geometry can be densified by adding interpolated points at regular intervals. Generalizing multipart geometry simplifies it while preserving its general shape. Densifying a multipart geometry adds more vertices at regular intervals.
 
-In this example, points representing a ships location are shown at irregular intervals. One can densify the polyline connecting these lines to interpolate points along the line at regular intervals. Generalizing the polyline can also simplify the geometry while preserving its general shape.
+![Image of densify and generalize](DensifyAndGeneralize.gif)
 
-![](DensifyAndGeneralize.gif)
+## Use case
 
-## How to use the sample
+The sample shows a polyline representing a ship's location at irregular intervals. The density of vertices along the ship's route is appropriate to represent the path of the ship at the sample map view's initial scale. However, that level of detail may be too great if you wanted to show a polyline of the ship's movement down the whole of the Willamette river. Then, you might consider generalizing the polyline to still faithfully represent the ship's passage on the river without having an overly complicated geometry.
 
-Use the sliders to control the parameters of the densify and generalize methods. You can deselect the checkboxes for either method to remove its effect from the result polyline. You can also hide the result to only see the original by deselecting the "Show result" checkbox.
+Densifying a multipart geometry can be used to more accurately represent curved lines or to add more regularity to the vertices making up a multipart geometry.
 
-## How it works
+## How to use the sample 
+
+Use the sliders to control the parameters of the densify and generalize methods. You can deselect the checkboxes for either method to remove its effect from the result polyline.
```

Comment:
We should keep in the part about the "Show result" part of the UI

Reply:
Agreed, I've added it back in.